actors/builtin/miner/miner_test.go

```diff
@@ -365,7 +367,9 @@ func TestWindowPost(t *testing.T) {
 		actor.constructAndVerify(rt)
 		st := getState(rt)
 		store := rt.AdtStore()
-		partitionSize := st.Info.WindowPoStPartitionSectors
+		info, err := st.GetInfo(store)
+		require.NoError(t, err)
+		partitionSize := info.WindowPoStPartitionSectors
```

Comment:
This is inconvenient enough that we should just move it to a method on the harness